less/global.less

```diff
@@ -4,6 +4,10 @@
   box-sizing: border-box;
 }
 
+html {
```

Comment:
Very good catch setting this global variable here! Many others missed this key concept.